packages/next-dashboard/src/components/FullscreenExitButton.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import React, { useContext, useEffect, useState } from 'react';
+import classnames from 'classnames';
+
+import LayoutContext from '../utils/layoutContext';
+
+export default function FullscreenExitButton(): React$Element<'div'> | null {
+  const ctx = useContext(LayoutContext);
+  const isFullscreen = ctx.getTemp('isFullscreen', false);
+  const [timeoutID, setTimeoutID] = useState<TimeoutID | void>();
+  const isMoving = ctx.getTemp('fullscreen-cursor-moving', false);
+  useEffect(() => {
+    if (typeof document === 'undefined' || !isFullscreen) {
+      return undefined;
+    }
+    let localTimeoutID = timeoutID;
+    let localIsMoving = isMoving;
+    const handleMouseMove = () => {
+      if (localTimeoutID) {
+        clearTimeout(localTimeoutID);
+      }
+      localTimeoutID = setTimeout(() => {
+        ctx.setTemp('fullscreen-cursor-moving', false);
+        localIsMoving = false;
+      }, 1500);
+      if (!localIsMoving) {
+        ctx.setTemp('fullscreen-cursor-moving', true);
+        localIsMoving = true;
+      }
+    };
+    document.addEventListener('mousemove', handleMouseMove);
+    return () => {
+      setTimeoutID(localTimeoutID);
+      document.removeEventListener('mousemove', handleMouseMove);
+    };
+  }, [isFullscreen, setTimeoutID]);
+  if (!isFullscreen) return null;
+
+  const handleClick = () => {
+    const contentEl =
+      typeof document !== 'undefined' && document.firstElementChild;
+    if (contentEl) document.exitFullscreen();
+  };
+
+  return (
+    <div
+      className={classnames(
+        'fullscreen-exit-button',
+        !isMoving && 'display-none',
+      )}
+    >
+      <button onClick={handleClick} title="Fullscreen mode" type="button">
+        Avsluta helskärm <span className="fa fa-compress-alt" />
```

Comment:
Considering the rest of next-dashboard is in english by default, it feels a bit weird to use swedish here. (even if all the projects we're currently working on are intended to be in swedish, it feels pretty inconsistent to mix languages in next-dashboard)

Reply:
This is a text string that will be visible to clients currently (clients who tend to complain about English in the dev site)

My suggestion is we leave it as is for now, and figure out how to deal with text-strings in next-dashboard which needs to be configurable on a project-by-project basis since we don't want to lock the entire project into a specific language?

Maybe you can provide a T function to the framework and it will run all strings through that?

Reply:
That sounds like a reasonable approach to me (and with `t` defaulting to identity).

But yeah, I'm fine with just keeping it for now I guess, feel free to mark as resolved